src/main/java/apps/Main.java

```diff
@@ -1,13 +1,72 @@
 package apps;
 
-import entities.User;
+import dao.PostgresCartDAO;
+import dao.PostgresGoodDAO;
+import entities.Good;
 import lombok.val;
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, EPAM!");
-        val user = new User();
-        user.setId(3);
-        System.out.println(user.getId());
+        //testGoodDao();
+        testReserveDao();
+    }
+
+    public static void testGoodDao() {
+        val goodDAO = new PostgresGoodDAO();
```

Comment:
What type you assigned to this variable? according to lombok manual it will be PostgresGoodDAO, but you decided to wrap DAOs in interfaces, isn't it?